tests/release/release.go

```diff
@@ -116,7 +116,7 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 				}
 
 				return strings.Contains(prList.Items[0].Name, releaseName)
-			}, 1*time.Minute, defaultInterval).Should(BeTrue())
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
 		It("The PipelineRun should exist and succeed.", func() {
```

Comment:
The _It_ block description does not comply with the format. Try to read it as it was not code: "It The Pipeline should exist and succeed" - the phrase does not make sense. Please rephrase it.

Reply:
Since I relate specifically to a PipelineRun, I need need to relate to the specific object used